.github/workflows/ci-systemd-release.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
       - uses: actions/setup-go@v3
         with:
-          go-version: "v1.21"
```

Comment:
Let's use go version from go.mod file

https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file
This will avoid us needing to update workflows all the time